include/seastar/core/seastar.hh

```diff
@@ -47,6 +47,7 @@
 /// View the [Seastar compatibility statement](./md_compatibility.html) for
 /// information about library evolution.
 
+#include <grp.h>
```

Comment:
we prefer not putting the 3rd-party headers in the purview when C++ modules are enabled. as the seastar C++module is not supposed to export the symbols provided by the 3rd-party headers . so could you please move this `#include` statement down into the `#ifndef SEASTAR_MODULE` block. you could put it after `#include <string_view>`, as `<string_view>` is a header from the standard library, and in `src/core/reactor.cc`, we put the 3rd party (the system library) headers after the headers from the standard library.

Reply:
Thanks for the explanation, when I added the <grp.h> within the `#ifndef SEASTAR_MODULE`, the compile using cxx modules was complaining about `struct group` being considered as `struct seastar::group`. I added a forward declaration `struct group` before the `namespace seastar` in this file and the compile using cxx modules passed.